osu.Game/Skinning/LegacySkin.cs

```diff
@@ -344,6 +348,37 @@ private void load(ISkinSource skin)
             }
         }
 
+        public class LegacySliderBall : CompositeDrawable
+        {
+            private readonly Drawable animationContent;
+
+            public LegacySliderBall(Drawable animationContent)
+            {
+                this.animationContent = animationContent;
+            }
+
+            [BackgroundDependencyLoader]
+            private void load(ISkinSource skin, DrawableHitObject drawableObject)
+            {
+                animationContent.Colour = skin.GetValue<SkinConfiguration, Color4?>(s => s.CustomColours.ContainsKey("SliderBall") ? s.CustomColours["SliderBall"] : (Color4?)null) ?? Color4.White;
+
+                InternalChildren = new[]
+                {
+                    new Sprite
```

Comment:
This and the spec texture don't seem to be working with the stable default skin during gameplay.

Reply:
they won't. they aren't included in the skin.

Reply:
Oh, ok then.